docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -261,15 +292,15 @@ If both `voter` and `follower` are defined in the rules, the replicas of `follow
 
 ```sql
 ALTER TABLE test
-	ADD PLACEMENT POLICY CONSTRAINTS="[+zone=bj]" ROLE=follower REPLICAS=2,
-	ALTER PLACEMENT POLICY CONSTRAINTS="[+zone=sh]" ROLE=voter REPLICAS=2;
+	PLACEMENT CONSTRAINTS="[+zone=bj]" ROLE=follower REPLICAS=2,
+	PLACEMENT CONSTRAINTS="[+zone=sh]" ROLE=voter REPLICAS=2;
```

Comment:
```suggestion
	CONSTRAINTS="[+zone=sh]" ROLE=voter REPLICAS=2;
```

Reply:
Do we still support `voter`?

Reply:
Yes. I don't have intentions to change this.